packages/host/tests/integration/search-index-test.ts

```diff
@@ -1165,8 +1173,24 @@ module('Integration | search-index', function (hooks) {
     }
   });
 
-  test('search doc only includes used fields', async function (assert) {
+  test(`search doc includes 'contains' and used 'linksTo' fields`, async function (assert) {
```

Comment:
could we also do a test where the linksTo field is not in the template (and not used), and since the linksTo field is not used it will not be part of the search doc